src/Events.Functions/Clients/InboundClient.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Altinn.Common.AccessTokenClient.Services;
+using Altinn.Platform.Events.Functions.Clients.Interfaces;
+using Altinn.Platform.Events.Functions.Configuration;
+using Altinn.Platform.Events.Functions.Extensions;
+using Altinn.Platform.Events.Functions.Models;
+using Altinn.Platform.Events.Functions.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Functions.Clients
+{
+    /// <summary>
+    /// Client used to send cloudEvents to events-inbound queue.
+    /// </summary>
+    public class InboundClient : SecureClientBase, IInboundClient
+    {
+        private readonly ILogger<IInboundClient> _logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InboundClient"/> class.
+        /// </summary>
+        public InboundClient(
```

Comment:
There isn't much logic in either of these clients.. Did you consider only having a single eventsClient that exposes a function for sending a request to each of the relevant endpoints?